README.md

```diff
@@ -1,37 +1,79 @@
-<h1 align="center">exasol/developer-documentation</h1>
+<h1 align="center">Developer Documentation</h1>
 
 <p align="center">
-Documentation and resources for developing software and data science solutions and applications on top of the Exasol Analytics Engine.
+Documentation and resources for data scientists and programmatic users to perform analytics with Exasol and to build applications on top.
 </p>
 
-<p align="center">
 
-<a href="https://opensource.org/licenses/MIT">
-    <img src="https://img.shields.io/pypi/l/exasol/developer_documentation" alt="License">
-</a>
-<a href="https://pypi.org/project/exasol/developer_documentation/">
-    <img src="https://img.shields.io/pypi/dm/exasol/developer_documentation" alt="Downloads">
-</a>
-<a href="https://pypi.org/project/exasol/developer_documentation/">
-    <img src="https://img.shields.io/pypi/pyversions/exasol/developer_documentation" alt="Supported Python Versions">
-</a>
-<a href="https://pypi.org/project/exasol/developer_documentation/">
-    <img src="https://img.shields.io/pypi/v/exasol/developer_documentation" alt="PyPi Package">
-</a>
-</p>
+# Getting Started
+There are three ways to get started with Exasol:
+- Download the [community edition](https://www.exasol.com/free-signup-community-edition/)
+- Use the [docker version](https://github.com/exasol/docker-db)
+- Signup for a free [SaaS trial](https://cloud.exasol.com/signup)
+
+In the following we are going to use the SaaS version.
+
+## Setting up the db
+Our [main documentation](https://docs.exasol.com/saas/get_started/saas_first_steps.htm) shows you how to create a SaaS account and setup your first database and cluster.
+## Installing pyexasol
+Once your database is up and running you can connect via any client. From Python we use [PyExasol](https://github.com/exasol/pyexasol).
+It can be easily installed via
+```
+pip install pyexasol[pandas]
+```
 
-## 🚀 Features
+## Connecting to the db
+Connecting to an Exasol database is performed via:
+```python
+import pyexasol
 
-## 🔌️ Prerequisites
+C = pyexasol.connect(dsn='<host:port>', user='sys', password='exasol')
+```
+It is slightly different when connecting to a SaaS database as we need a [personal access token](https://docs.exasol.com/saas/administration/access_mngt/access_token.htm):
+```python
+import pyexasol
 
-- [Python](https://www.python.org/) >= 3.9
+C = pyexasol.connect(dsn='<host:port>', user='sys', refresh_token='<token>')
```

Comment:
no, you can use the PAT simply as password, we fixed that in the pyexasol documentation, lately.

```suggestion
C = pyexasol.connect(dsn='<host:port>', user='sys', password='<token>')
```